components/editor/modules/link/ui.js

```diff
@@ -140,6 +159,10 @@ export default ({TYPE}) => {
                   onChange={onInputChange('title')}
                 />
                 <SearchUserForm onChange={authorChange(onChange, value, node)} />
+                <RepoSearch
+                  label={t('link/repo/search', undefined, 'Artikel suchen')}
```

Comment:
why the default? Please add it to the sheet https://docs.google.com/spreadsheets/d/1qPubCKz2YzbcFkRfF0MrgRqD0KLsgRPxoZfG5G3tHOA/edit#gid=0 and run `npm run translations`